SU2_CFD/include/driver_structure.hpp

```diff
@@ -413,63 +413,60 @@ class CMultiZoneDriver : public CDriver {
 
 /*!
  * \class CSpectralDriver
- * \brief Class for driving an iteration of a spectral method problem using multiple zones.
+ * \brief Class for driving an iteration of harmonic balance method problem using multiple zones.
```

Comment:
Thanks Tom. Yes, we actually discussed about this together with Sravya. I changed to CHBDriver now. I used CHBDriver instead of CHarmonicBalanceDriver thinking at the next implementation that would be something like CMultiZoneHB. Let me know what you think about it.